Document/0x05g-Testing-Network-Communication.md

```diff
@@ -250,10 +294,16 @@ Dynamic analysis can be performed by launching a MITM attack with your preferred
 #### Overview
 Network Security Configuration was introducted on Android 7 and lets apps customize their network security settings such as custom trust anchors and Certificate pinning.
 
-When apps target API Levels 24+ and are running on an Android device with versions 7+, they used a default Network Security Configuration that doest not trust user supplied CA's, reducing the possibility of MiTM attacks by luring users to install malicious CA's.
+##### Trust Anchors
+
+When apps target API Levels 24+ and are running on an Android device with versions 7+, they use a default Network Security Configuration that doest not trust user supplied CA's, reducing the possibility of MiTM attacks by luring users to install malicious CA's.
 
 This protection can be bypassed by using a custom Network Security Configuration with a custom trust anchor indicating that the app will trust user supplied CA's.
 
+##### Pin-set Expiration Date
+
+Pin-set contain a set of public key pins. Each set can define a expiration date. When the expiration date is reached, the network communication will continue to work, but the Certificate Pinning will be disabled for the affected domains. Be aware of this behavior.
```

Comment:
This seems to be a bit repetitive with the previous part, perhaps you can reference to it?

Perhaps we can remove the last sentence?

Reply:
Please only describe it once. Might be best to only describe it here, as this is the test case for network security config, therefore it should be a bit more detailed.

Reply:
Fixed.